packages/create-nx-workspace/.eslintrc.json

```diff
@@ -33,7 +33,7 @@
           "error",
           {
             "buildTargets": ["build-base"],
-            "ignoredDependencies": []
+            "ignoredDependencies": ["nx"]
```

Comment:
is this needed?

Reply:
Turns out to be needed just from the dynamic require,